src/main/java/frc/team2412/robot/commands/intake/IntakeBallCommand.java

```diff
@@ -0,0 +1,21 @@
+package frc.team2412.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import frc.team2412.robot.commands.index.IndexCommand;
+import frc.team2412.robot.subsystem.IndexSubsystem;
+import frc.team2412.robot.subsystem.IntakeSubsystem;
+
+public class IntakeBallCommand extends ParallelCommandGroup {
+    private final IndexSubsystem indexSubsystem;
+    private final IntakeSubsystem intakeSubsystem;
+
+    public IntakeBallCommand(IndexSubsystem indexSubsystem, IntakeSubsystem intakeSubsystem) {
+        this.indexSubsystem = indexSubsystem;
+        this.intakeSubsystem = intakeSubsystem;
+        addRequirements(indexSubsystem, intakeSubsystem);
+        addCommands(
+                new IntakeExtendCommand(intakeSubsystem),
+                new IntakeInCommand(intakeSubsystem),
```

Comment:
For this command and `SpitBallCommand`: Running two commands that both require the same subsystem (intake in this case) in a `ParallelCommandGroup` will throw an exception because the two commands would run at the same time and conflict with each other.